lang/en/logstore_xapi.php

```diff
@@ -98,3 +98,12 @@
 $string['privacy:metadata:logstore_xapi_log:userid'] = 'User Id of xAPI holding table for cron processing';
 $string['privacy:metadata:logstore_xapi_failed_log'] = 'xAPI holding table for failed events';
 $string['privacy:metadata:logstore_xapi_failed_log:userid'] = 'User Id of xAPI holding table for failed events';
+
+// Info strings from xAPI errors
+$string['networkerror'] = 'There was a network error sending the response: {$a}';
+$string['recipeerror'] = 'The LRS responded with a 400 error, this can be due to an issue with the recipe. The following message
+    was returned: "{$a}"';
+$string['autherror'] = 'The server is returning a 401 error: {$a}. Please ensure the endpoint, username and auth secret/password for
+    the xAPI is correct in the Logstore xAPI settings.';
+$string['unknownerror'] = 'Error code: "{$a->errortype}" Response: "{$a->response}"';
+$string['lrserror'] = 'There is a problem with the LRS. The LRS has responded with a 500 error. Response: {$a}';
```

Comment:
JISC refer to an LRS as LDH - we should either change these strings, or allow add an additional parameter which picks up the value from another lang string

Reply:
We have to be careful with JISC specific changes and what the community of users would expect. Is LDH a JISC specific term?

Reply:
I believe so @mlynn-lp

Reply:
That's why I left LRS in the key for the strings, by default these will be LDH (Learner Data Hub) but this can always be changed using the language pack. As this is the JISC branch, I don't see the issue really with sticking with LDH